frontend/src/components/common/SnackbarAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
I believe we use Toasts for warnings on the student side. Is this something different or could we maybe use the same Toasts?

Reply:
I believe it's the same functionality but I think it would be nice if we don't have to add in all of the useToast/toastProvider stuff that was necessary on the student end. Also if I recall correctly we have a bit of a mix between snackbar and toast being used on the students end (undo button and delete plan is snackbar and warnings are Toast)